templates/it/docs/overview.md

```diff
@@ -0,0 +1,52 @@
+<% seoDescription("Buffalo framework overview") %>
+<% seoKeywords(["buffalo", "framework", "overview", "go", "golang", "mux", "bootstrap", "jquery"]) %>
+
+<%= h1("Overview") %>
```

Comment:
This is the title of the page, can you translate it too? :)

Reply:
Sure. Did it